### PR TITLE
add sample image when logs are not present

### PR DESCRIPTION
**Pull request by svilenko**
add sample image when logs are not present

image

### CODE REVIEW

File: Workbooks/ADXCluster/AtResourceClusterBoundaries/ResourceClusterBoundaries.workbook [hunk 1]
```diff
@@ -1340,7 +1297,7 @@
                   "type": 3,
                   "content": {
                     "version": "KqlItem/1.0",
-                    "query": "AzureMetrics\r\n| where MetricName == \"CacheUtilization\"\r\n| project TimeGenerated, CacheUtilization=Average\r\n| summarize CacheUtilization=avg(CacheUtilization) by bin(TimeGenerated, 1d)\r\n| sort by TimeGenerated asc \r\n| serialize \r\n| extend PrevCacheUtilization = prev(CacheUtilization,1)\r\n| extend CacheUtilizationDelta = CacheUtilization - PrevCacheUtilization\r\n| project TimeGenerated, CacheUtilizationDelta\r\n| extend change=iff(CacheUtilizationDelta >= 0, \"Positive\", \"Negative\")\r\n",
+                    "query": "AzureMetrics\r\n| where MetricName == \"CacheUtilization\"\r\n| project TimeGenerated, CacheUtilization=Maximum\r\n| summarize CacheUtilization=avg(CacheUtilization) by bin(TimeGenerated, 1d)\r\n| sort by TimeGenerated asc \r\n| serialize \r\n| extend PrevCacheUtilization = prev(CacheUtilization,1)\r\n| extend CacheUtilizationDelta = CacheUtilization - PrevCacheUtilization\r\n| project TimeGenerated, CacheUtilizationDelta\r\n| extend change=iff(CacheUtilizationDelta >= 0, \"Positive\", \"Negative\")\r\n",
```
Comment (liatbezalel) [inline Workbooks/ADXCluster/AtResourceClusterBoundaries/ResourceClusterBoundaries.workbook:1300]: why?

File: Workbooks/ADXCluster/AtResourceClusterBoundaries/ResourceClusterBoundaries.workbook [hunk 2]
```diff
@@ -23,49 +23,6 @@
...
-                },
-                {
-                  "durationMs": 259200000
-                },
-                {
-                  "durationMs": 604800000
-                }
-              ],
-              "allowCustom": false
-            }
```
Comment (liatbezalel) [inline Workbooks/ADXCluster/AtResourceClusterBoundaries/ResourceClusterBoundaries.workbook:68]: why?

Comment (svilenko) [reply Workbooks/ADXCluster/AtResourceClusterBoundaries/ResourceClusterBoundaries.workbook]: fixed- took changes from git

File: Workbooks/ADXCluster/AtResourceClusterBoundaries/ResourceClusterBoundaries.workbook (hunk 1 above)
Comment (svilenko) [reply Workbooks/ADXCluster/AtResourceClusterBoundaries/ResourceClusterBoundaries.workbook]: fixed - took changes from git

**Review by liatbezalel — APPROVED** [review]

**Comment by liatbezalel** [comment]
Pay attenation that the build is failing

**Pull request merged**